core/src/test/java/org/apache/cxf/helpers/FileUtilsTest.java

```diff
@@ -79,4 +78,15 @@ public void testGetFiles() throws URISyntaxException {
 
         assertEquals(foundFiles, foundFiles2);
     }
+
+    @Test
+    public void testStripPath() {
+        String name = "c:\\boo\foo\\file.txt";
```

Comment:
Please split into two test cases:
 - Windows style
 - Unix style

Reply:
@reda-alaoui  Done. It is much better to have very specific unit test